modules/motions/hooks/useRegistryWithLimits.ts

```diff
@@ -28,9 +28,9 @@ type AllowedRecipient = {
 export const REGISTRY_WITH_LIMITS_BY_MOTION_TYPE = {
   [MotionType.LegoLDOTopUp]: ContractLegoLDORegistry,
   [MotionType.LegoDAITopUp]: ContractLegoDAIRegistry,
-  [MotionType.RccDAITopUp]: ContractRccDAIRegistry,
-  [MotionType.PmlDAITopUp]: ContractPmlDAIRegistry,
-  [MotionType.AtcDAITopUp]: ContractAtcDAIRegistry,
+  [MotionType.RccDAITopUp]: ContractRccStablesRegistry,
```

Comment:
probaly should be old regestry 
```
  [MotionType.RccDAITopUp]: ContractRccDAIRegistry,
  [MotionType.PmlDAITopUp]: ContractPmlDAIRegistry,
  [MotionType.AtcDAITopUp]: ContractAtcDAIRegistry, 
```

Reply:
No, shouldn't be. New factories are working with old registries, so I think the name should be more generic.